src/partials/version-navigation.handlebars

```diff
@@ -0,0 +1,12 @@
+<div class="nav navbar-nav navbar-right">
+    <li>
+        <a href="/pim/serenity/index.html">Serenity</a>
+    </li>
+    <li id="last-EE-version" data-filepath="{{filePath}}" data-major-version="{{majorVersion}}"></li>
+    <li class="dropdown">
+        <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">v2 EE/CE <span class="caret"></span></a>
+        <ul class="dropdown-menu" id="version-dropdown" data-filepath="{{filePath}}" data-major-version="{{majorVersion}}">
```

Comment:
### 
```suggestion
        <ul class="dropdown-menu" id="version-dropdown" data-filepath="{{filePath}}" data-current-documenttion-version="{{currentDocumentationVersion}}">
```

Reply:
I struggled to understand the code due this naming I think

Reply:
```suggestion
        <ul class="dropdown-menu" id="version-dropdown" data-filepath="{{filePath}}" data-current-doc-version="{{majorVersion}}">
```